crates/networking/rpc/clients/mod.rs

```diff
@@ -3,4 +3,4 @@ pub mod beacon;
 pub mod eth;
 
 pub use auth::{EngineClient, errors::EngineClientError};
-pub use eth::{EthClient, errors::EthClientError, eth_sender::Overrides};
+pub use eth::{EthClient, errors::EthClientError};
```

Comment:
You can re-export `Overrides` here and restore all paths:
```suggestion
pub use eth::{EthClient, errors::EthClientError, Overrides};
```